### PR TITLE
Fix disappearing custom backend popup

### PR DESCRIPTION
**Pull request by marekrjpolak**
## Description

`WrapperDiv` didn't propagate `onClick` to underlying div so the custom backend popup closed immediately. Also `open` and `setOpen` were unused.

### CODE REVIEW

File: packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/QuickActions/NavBackends.tsx [hunk 1]
```diff
@@ -89,13 +89,20 @@ type NavBackendsProps = {
...
+const WrapperDiv = ({
+    children,
+    isDisabled: _, // `isDisabled` is needed here despite not being used
+    onClick,
+}: {
+    isDisabled?: boolean;
+    children: ReactNode;
+    onClick?: () => void;
+}) => (
+    // eslint-disable-next-line
```
Comment (jvaclavik) [inline packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/QuickActions/NavBackends.tsx:101]: Why is this necessary?

**Review by jvaclavik — APPROVED** [review]

File: packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/QuickActions/NavBackends.tsx (hunk 1 above)
Comment (marekrjpolak) [reply packages/suite/src/components/suite/layouts/SuiteLayout/Sidebar/QuickActions/NavBackends.tsx:101]: Because it's not recommended to pass onClick directly to div, at least without specifying some accessibility stuff.

**Comment by bosomt** [comment]
**QA OK**

**Info:**
 - Suite version: desktop 25.1.0 (20a35a51ea92edb7ffa42c696f6da297c66d2971)
 - Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) TrezorSuiteDev/25.1.0 Chrome/128.0.6613.186 Electron/32.2.6 Safari/537.36
 - OS: MacIntel
 - Screen: 1512x982
 - Device: Trezor T3T1 2.8.3 regular (revision 7f373ae71eca855dd41b4a0fdcc7cadaa13a8281)
 - Transport: BridgeTransport 2.0.33
